pom.xml

```diff
@@ -650,6 +647,17 @@
   </build>
 
   <profiles>
+    <profile>
+      <id>jdk9+</id>
+      <activation>
+        <jdk>[9,)</jdk>
+      </activation>
+      <properties>
+        <!-- release takes precedence over source/target if java version is 9 or higher -->
+        <maven.compiler.release>8</maven.compiler.release>
```

Comment:
Why putting 8 here?

Reply:
Instead of `1.8` you mean? Starting Java 9, version switched from 1.5, 1.6, 1.7 to 9, 10, 11. And the common practice is to use 8 although it may be possible that 1.8 is supported as well (but 1.9 would not work).